wcivf/templates/home.html

```diff
@@ -33,9 +29,9 @@ <h1 class="ds-h2 ds-text-centered" style="text-wrap: balance">
 
         </form>
     </div>
-    {% include "parl_24_hex_map.html" %}
+    {% comment %} {% include "parl_24_hex_map.html" %} {% endcomment %}
```

Comment:
This and the whole hex map thing can go, too. In fact, almost all of https://github.com/DemocracyClub/WhoCanIVoteFor/pull/1998/commits/2ab984ecd50877179f998fc3f09245bf96e36945 can be deleted!